kdl_parser_py/package.xml

```diff
@@ -19,16 +20,16 @@
   <url type="repository">https://github.com/ros2/kdl_parser</url>
   <url type="bugtracker">https://github.com/ros2/kdl_parser/issues</url>
 
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
-  <buildtool_depend>python-catkin-pkg</buildtool_depend>
-
   <build_depend version_gte="1.3.0">orocos_kdl</build_depend>
   <build_depend>urdf</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>urdf</run_depend>
-  <run_depend>urdfdom_py</run_depend>
-  <run_depend>python_orocos_kdl</run_depend>
+  <exec_depend version_gte="1.3.0">orocos_kdl</run_depend>
+  <exec_depend>urdf</run_depend>
+  <exec_depend>urdfdom_py</run_depend>
+  <exec_depend>python_orocos_kdl</run_depend>
```

Comment:
Unfortunately, this is going to be a problem.  We don't have `python_orocos_kdl` ported to ROS 2 at this time.  https://github.com/ros2/geometry2/pull/360#issuecomment-754651422 has some of my thoughts on how to go about enabling this.